src/redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BookList from '../../components/bookList';
+import AddNew from '../../components/addNew';
+
+const Books = () => (
+  <div className="container">
+    <div className="inner">
+      <BookList />
+      <AddNew />
+    </div>
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] There is a misunderstanding here. The book.js file in redux is for setup functionality for redux. It is not for your components. Please keep this file empty for this milestone and create your components in your components folder. ♻️